tests/test_report_biosample_slots_with_ucum_constraint.py

```diff
@@ -0,0 +1,68 @@
+import os
```

Comment:
code like this would have to included in both the runtime and in this repo's examples runner

integration with runtime needs lots more discussion and will require migrators for each way in which a has_unit could violate the specification from the annotations

Reply:
possibly use linkml-validator's extension hooks